Cargo.toml

```diff
@@ -74,17 +74,17 @@ minstant = "0.1.3"
 coverage-helper = "0.2.0"
 
 # workspace
-fvm = { path = "fvm", version = "~4.2.0", default-features = false }
-fvm_shared = { path = "shared", version = "~4.2.0", default-features = false }
-fvm_sdk = { path = "sdk", version = "~4.2.0" }
+fvm = { path = "fvm", version = "~4.2.1", default-features = false }
+fvm_shared = { path = "shared", version = "~4.2.1", default-features = false }
+fvm_sdk = { path = "sdk", version = "~4.2.1" }
```

Comment:
Should be 4.3 given the proofs version bump (which, IIRC, forces proofs to be updated everywhere else).

Also note: I _think_ we may need to update proofs in FVM v2 and v3 as well.